docs/book/src/design/wallet-connector-app.md

```diff
@@ -0,0 +1,7 @@
+# Wallet Connector app
+
+The Wallet Connector app runs on web and smartphone (Android, iOS) devices, and allows users other than the device owner to create and access their wallets, using a combination of QR code and PIN based authentication.
```

Comment:
- Somewhere, it should probably be made explicit that the *Connector app* is the client application
- This paragraph seems to suggest that parties other than the wallet owner has access to the funds of the latter

Reply:
Good catch. How about this?

```suggestion
The Wallet Connector app runs on web and smartphone (Android, iOS) devices, and allows transacting users to manage wallets and authorise transactions using a combination of QR codes and PIN based authentication.

The Connector app is primarily intended to facilitate transactions for transacting users other than the device owner (such as customers, friends, or family members, friends, or customers),
so each session that the Connector app establishes with the Wallet TEE is designed to be interactive, private, and short-lived. 
The Connector app retains no session state between sessions, and device owners and different transacting users have no access to each others' wallets or accounts.
```

Reply:
I think this eliminates any ambiguity yes :+1: